synapse/config/registration.py

```diff
@@ -152,6 +156,10 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
         #
         #session_lifetime: 24h
 
+        # MSC2918
+        # TODO: docs
+        #access_token_lifetime: 5m
+
```

Comment:
we should maybe leave this out of the documentation altogether until MSC2918 is formalised, tbh.

```suggestion
```